res/css/views/dialogs/_DevtoolsDialog.scss

```diff
@@ -160,34 +157,34 @@ limitations under the License.
             border-radius: 4px;
         }
 
-        &:after {
+        &::after {
             content: attr(data-tg-on);
-            background: #02C66F;
+            background: #02c66f;
```

Comment:
this kind of change is a bit silly to enforce imo

Reply:
Ah yeah, perhaps so... At the moment, we're mostly just using the `stylelint-config-standard` rule set, but we could tweak individual rules if we want to.